code/game/machinery/computer/camera_advanced.dm

```diff
@@ -65,6 +63,14 @@
 		give_eye_control(L)
 		eyeobj.setLoc(eyeobj.loc)
 
+/obj/machinery/computer/camera_advanced/attack_robot(mob/user)
+	if(!Adjacent(user)) //Borgs can use the console without issue so long as they remain next to it.
```

Comment:
this is dumb, the vast majority (all?) borg->machine interactions are wireless+rangeless
if you're worried about powergaming, limit it to the same screen (```get_dist(src, user) <= user.client.view``` (obviously add some safeties))